backend/session/session.c

```diff
@@ -263,6 +263,11 @@ size_t wlr_session_find_gpus(struct wlr_session *session,
 		return explicit_find_gpus(session, ret_len, ret, explicit);
 	}
 
+#ifdef __FreeBSD__
+	// XXX: libudev-devd does not return any GPUs (yet?)
```

Comment:
Did you file a bug to follow up on this?

Reply:
There's no issue tracker on https://github.com/FreeBSDDesktop/libudev-devd — these projects aren't taken as seriously as you think :)

Reply:
They're still overseen by someone. Maybe there's a parent project or a mailing list or if all else fails you can just email some recent committers directly.